envs/monkey_zoo/blackbox/analyzers/performance_analyzer.py

```diff
@@ -0,0 +1,50 @@
+import logging
+from datetime import timedelta
+
+from envs.monkey_zoo.blackbox.analyzers.analyzer import Analyzer
+from envs.monkey_zoo.blackbox.island_client.monkey_island_client import MonkeyIslandClient
+
+MAX_ALLOWED_SINGLE_PAGE_TIME = timedelta(seconds=2)
+MAX_ALLOWED_TOTAL_TIME = timedelta(seconds=5)
+
+logger = logging.getLogger(__name__)
+
+
+class PerformanceAnalyzer(Analyzer):
+
+    def __init__(self, island_client: MonkeyIslandClient, break_if_took_too_long=False):
+        self.break_if_took_too_long = break_if_took_too_long
+        self.island_client = island_client
+
+    def analyze_test_results(self) -> bool:
+        if not self.island_client.is_all_monkeys_dead():
+            logger.info("Can't test report times since not all Monkeys have died.")
+            return False
```

Comment:
Exception would be better, because 1. exceptions > return codes 2. Analysis neither failed nor succeeded, it wasn't ran.

Reply:
Correct, this is a leftover from when this was a regular analyzer and not a post_exec analyzer. Fixed